src/tests/unit/components/search-filter-input/SearchFilterInput.spec.jsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import SearchFilterInput from '~/components/search-filter-input/SearchFilterInput'
+import { vi } from 'vitest'
```

Comment:
First import libs and after components
```suggestion
import { vi } from 'vitest'
import { render, screen, fireEvent } from '@testing-library/react'
import SearchFilterInput from '~/components/search-filter-input/SearchFilterInput'
```